src/services/OswServiceTaskWiFi.cpp

```diff
@@ -214,7 +270,7 @@ bool OswServiceTaskWiFi::isStationEnabled() {
  * @param password Set the wifi password to this (at least 8 chars!), otherwise a random password will be choosen.
  */
 void OswServiceTaskWiFi::enableStation(const String& password) {
```

Comment:
add `this->m_connectFailureCount = 0;`?

Reply:
No, as this will has nothing to do with the connection to an other station. This is creating a new station on its own and must therefore not influence the client mode.